test/integration/ws-test.js

```diff
@@ -0,0 +1,40 @@
+let sandbox = require('@architect/sandbox')
+let test = require('tape')
+let { join } = require('path')
+let WebSocket = require('ws')
+
+let port = process.env.PORT ? process.env.PORT : '3333'
+let url = `ws://localhost:${port}`
+
+let mock = join(__dirname, '..', 'mock', 'project')
+let origCwd = process.cwd()
+
+test('Set up env', async t => {
+  t.plan(2)
+  process.chdir(mock)
```

Comment:
fyi we should never need to do this in tests anymore, Sandbox accepts a `cwd` option!